bin/messageformat.js

```diff
@@ -57,7 +56,14 @@ function printUsage() {
     '        The global object or modules format for the output JS. If _ns_ does not',
     '        contain a \'.\', the output follows an UMD pattern. For module support,',
     '        the values \'*export default*\' (ES6), \'*exports*\' (CommonJS), and',
-    '        \'*module.exports*\' (node.js) are special. [default: *module.exports*]'
+    '        \'*module.exports*\' (node.js) are special. [default: *module.exports*]',
+    '',
+    '  *--disable-plural-key-checks*',
+    '        Previous versions of messageformat.js allowed the use of plural &',
+    '        selectordinal statements with any keys; now we throw an error when a',
+    '        statement uses a non-numerical key that will never be matched as a',
+    '        pluralization category for the current locale. Use this argument to',
+    '        disable the validation and allow usage as previously. [default: *false*]'
```

Comment:
The "previous versions" bit is extraneous here. Replace with "By default, messageformat.js throws an error when...'